ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -25,13 +23,54 @@ import { invenioConfig } from '@inveniosoftware/react-invenio-app-ils';
 export class ImportedDocuments extends React.Component {
   constructor(props) {
     super(props);
-    this.pageSize = 100;
     this.state = {
       importCompleted: false,
       data: null,
       isLoading: true,
-      activePage: 1,
+      selectedResult: 'records',
+      statistics: {
+        records: {
+          text: 'Records',
+          value: '',
+          filterFunction: a => a,
+          filteredRecords: [],
+        },
+        records_created: {
+          text: 'Records created',
+          value: '',
+          filterFunction: a => a.action === 'create',
```

Comment:
Let's avoid naming variables `a`, `b`, `c` if possible ;)

Reply:
I agree, explicit naming would be more readable